pkgs/applications/misc/logseq/default.nix

```diff
@@ -4,7 +4,7 @@
 , appimageTools
 , makeWrapper
 # graphs will not sync without matching upstream's major electron version
```

Comment:
Should we move this comment to `all-package.nix` now? I think it'll be most useful where the version is being pinned.

Reply:
I overlooked that one, but it's fixed now. Thank you!